scripts/test.py

```diff
@@ -79,12 +79,41 @@ def monte_carlo_sampling_scipy(N_FEATURES, SAMPLES, centered=False, strength=2,
     Documentation for Latin Hypercube: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.qmc.LatinHypercube.html#scipy.stats.qmc.LatinHypercube
     Orthogonal LHS is better than basic LHS: https://github.com/scipy/scipy/pull/14546/files, https://en.wikipedia.org/wiki/Latin_hypercube_sampling
     """
-    from scipy.stats import qmc    
+    from scipy.stats import qmc, norm, lognorm, beta, gamma, triang
+    from sklearn.preprocessing import MinMaxScaler
     
     sampler = qmc.LatinHypercube(d=N_FEATURES, centered=centered, strength=strength, optimization=optimization, seed=seed)
     lh = sampler.random(n=SAMPLES)
     discrepancy = qmc.discrepancy(lh)
     print("Discrepancy is:", discrepancy, " more details in function documentation.")
+    
+
+    if DISTRIBUTION == "Uniform":
+        pass
+    elif DISTRIBUTION == "Normal":
+        mean, std= DISTRIBUTION_PARAMS
+        lh = norm.ppf(lh, mean, std)
+    elif DISTRIBUTION == "LogNormal":
+        mean, std = DISTRIBUTION_PARAMS
+        lh = lognorm.ppf(lh, s=0.90)
+    elif DISTRIBUTION == "Triangle":
+        tri_mean = np.mean(DISTRIBUTION_PARAMS)
+        lh = triang.ppf(lh, tri_mean)
+    elif DISTRIBUTION == "Beta":
+        if np.min(DISTRIBUTION_PARAMS) == 0:
+            raise ValueError(f"{DISTRIBUTION} parameters cannot be less than or equals to zero")
+        a, b = DISTRIBUTION_PARAMS
+        lh = beta.ppf(lh, a, b)
+    elif DISTRIBUTION == "Gamma":
+        if np.min(DISTRIBUTION_PARAMS) == 0:
+            raise ValueError(f"{DISTRIBUTION} parameters cannot be less than or equals to zero")
```

Comment:
The checks seems to completely duplicate those implemented in #2. So, the suggestion is to wrap-up checks in #2 into a function and use it regardless of a particular way a distribution is being generated. What do you think?

Reply:
Thanks for the review. I will work on the comments and revert back to you.